test/unit/evault/modules/Vault/loopDeloop.t.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
let's merge this whole test with `deloop` if possible. I think best to keep `loopDeloop`